README.md

```diff
@@ -96,6 +96,10 @@ group are derived from username given in `ssh_user` variable.
 
 When set to *false*, the original `ssh_config` file is not backed up. Default is *true*.
 
+### ssh_transactional_update_reboot_ok
+
+This variable is used to handle reboots required by transactional updates. If a transactional update requires a reboot, the role will proceed with the reboot if ssh_transactional_update_reboot_ok is set to true. If set to false, the role will notify the user that a reboot is required, allowing for custom handling of the reboot requirement. If this variable is not set, the role will fail to ensure the reboot requirement is not overlooked.
```

Comment:
can you wrap the long lines similarly as the rest of the readme? These long lines make the diffs quite useless and paragraph hard to read on systems that do not wrap automatically.